janus.go

```diff
@@ -107,6 +229,34 @@ func passMsg(ch chan interface{}, msg interface{}) {
 	ch <- msg
 }
 
+func (gateway *Gateway) longPoll() {
```

Comment:
How does this function get cancelled?

Reply:
nothing yet, I'll add a close ch for when the sessions aren't active anymore